linker.go

```diff
@@ -121,7 +121,7 @@ func fixupJumpsAndCalls(insns asm.Instructions) error {
 
 			ins.Constant = int64(callOffset - offset - 1)
 
-		case ins.OpCode.Class() == asm.JumpClass && ins.Offset == -1:
+		case ins.OpCode.JumpOp() != asm.InvalidJumpOp && ins.Offset == -1:
```

Comment:
I find this harder to read. How about assing `Class.IsJump()` and then doing the following:

```suggestion
		case ins.OpCode.Class().IsJump() && ins.Offset == -1:
```

We could apply `IsJump`, `IsALU` in other places as well.

Reply:
Good idea! Added a commit to introduce these, and `IsStore()`, `IsLoad()` as well.